### PR TITLE
Update PowerMockAgent.java

### PR DESCRIPTION
**Pull request by hsynkrtl**
added java9 to jdk6OrLater

### CODE REVIEW

File: powermock-modules/powermock-module-javaagent/src/main/java/org/powermock/modules/agent/PowerMockAgent.java [hunk 1]
```diff
@@ -37,7 +37,8 @@
 {
 
 	static final String javaSpecVersion = System.getProperty("java.specification.version");
-    static final boolean jdk6OrLater = "1.6".equals(javaSpecVersion) || "1.7".equals(javaSpecVersion) || "1.8".equals(javaSpecVersion);
+    static final boolean jdk6OrLater = "1.6".equals(javaSpecVersion) || "1.7".equals(javaSpecVersion) || "1.8".equals(javaSpecVersion)
+	    || "9".equals(javaSpecVersion);
```
Comment (hsynkrtl) [inline powermock-modules/powermock-module-javaagent/src/main/java/org/powermock/modules/agent/PowerMockAgent.java:41]: java 9 specification version key added

**Comment by mkmaier** [comment]
This will fail for java 10 and later. If the check is inverted, though, it should work for any new version, without changes. So I'd suggest something like

    static final boolean jdk6OrLater = !("1.4".equals(javaSpecVersion) || "1.5".equals(javaSpecVersion))

**Comment by rusefi** [comment]
So what about version "10" or "11-ea" or "11"? :(

**Comment by thekingn0thing** [comment]
I can say that supporting java 10,11 can happen early than beginning of next year only in case if someone are ready to provide a pull request with fix and confirmation for Travis.

If builds with tests are passed on Java 10, 11 I’m happy to merge the pull request and make a release.

Thanks,
Artur

On 24 Jul 2018, at 22:18, rusefi <notifications@github.com<mailto:notifications@github.com>> wrote:


So what about version "10" or "11-ea" or "11"? :(

**Pull request merged**